app/javascript/flavours/polyam/styles/homogay/diff.scss

```diff
@@ -254,10 +254,21 @@
 
 // settings
 
+// Polyam: Keep background in admin UI a bit darker
+body.admin {
+  --background-color: #{darken($ui-base-color, 4%)};
+}
+
 .admin-wrapper {
   .sidebar ul .simple-navigation-active-leaf a,
   .content__heading__tabs a.selected {
-    background-color: $ui-highlight-color;
     text-shadow: 0 0 6px $faint-shadow-color;
```

Comment:
The text-shadow is no longer needed and pretty much invisible, but it's a nice detail.

Reply:
Decided to remove it anyway.